jdaviz/configs/default/plugins/data_quality/tests/test_data_quality.py

```diff
@@ -145,7 +145,7 @@ def test_data_quality_plugin(imviz_helper, tmp_path):
     label_mouseover._viewer_mouse_event(viewer,
                                         {'event': 'mousemove', 'domain': {'x': 1371, 'y': 715}})
     label_mouseover_text = label_mouseover.as_text()[0]
-    assert label_mouseover_text.split('+')[1] == '2.94744e-01 MJy/sr (DQ: 4)'
+    assert label_mouseover_text.split('+')[1] == '2.94834e-01 MJy/sr (DQ: 4)'
```

Comment:
p.s. If we really don't care about the actual value, maybe we can try regex or just not check the value itself.